src/components/cards/boolean-card.tsx

```diff
@@ -120,19 +145,21 @@ export const BooleanCard = ({
             setIsOpen(true);
           }}
         >
-          {minVersion === "1.6.0" && <NewItemBadge version={minVersion}/>}
+          {minVersion === "1.6.0" && <NewItemBadge version={minVersion} />}
           <div
-            className={cn(
-              "flex items-center space-x-3 truncate text-left",
-              minVersion === "1.6.0" && !show && !completed && "blur-sm",
-            )}
+            className={
+              "flex items-center space-x-3 truncate text-left " +
+              (minVersion === "1.6.0" && !show && !completed && "blur-sm")
```

Comment:
Have minVersion be an env variable and not hardcoded.

Reply:
I like the idea of the most recently supported version being a variable. One option would be having it be an env variable and then blurring only most recent update content, another is that future patches get new items with new min-versions we can have a "min-version" setting per user that uses a semver lib to blur everything above the selected version.

In either case, this would be a non-trivial lift, and should probably be handled in a new PR where progress app-wide can be tracked.

Reply:
I will tackle this if desired on a seperate PR